### PR TITLE
doc(pendo): dbt model README edit for Fivetran style

### PR DESCRIPTION
**Pull request by fivetran-chloe**
Links https://fivetran.atlassian.net/browse/RD-129041

Edits for Fivetran style. Rewords for clarity. Please address my question (inline) and check my edits for technical accuracy.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,8 +5,8 @@ This package models Pendo data from [Fivetran's connector](https://fivetran.com/
 
 This package enables you to better understand how users are experiencing and adopting your product. It does so by:
 - Calculating usage of features, pages, guides, and the overall product at the account and individual visitor level
-- Enhancing event stream tables with visitor and product information, along with referring pages and features in order to cultivate a customer-journey through the application
-- Creating daily activity timelines for features, pages, and guides, reflecting their adoption rates, discoverability, and efficacy at promoting more usage
+- Enhancing event stream tables with visitor and product information, referring pages, and features to cultivate a customer journey through the application
```
[the remark this answers is not in the document]
Comment (fivetran-jamie) [reply README.md:8]: hm like discerning the customer journey better? basically the event stream tables now have the previous feature and/or page that the event-triggering user last visited, so with each event record, you know where the visitor came from

File: README.md [hunk 2]
```diff
@@ -5,20 +5,20 @@ This package models Pendo data from [Fivetran's connector](https://fivetran.com/
...
+- Creating daily activity timelines for features, pages, and guides that reflect their adoption rates, discoverability, and usage promotion efficacy
 - Directly visitors and features to determine activation rates, power-usage, and churn risk
 
 ## Models
 
-This package contains transformation models, designed to work simultaneously with our [Pendo source package](https://github.com/fivetran/dbt_pendo_source). A dependency on the source package is declared in this package's `packages.yml` file, so it will automatically download when you run `dbt deps`. The primary outputs of this package are described below. Intermediate models are used to create these output models.
+This package contains transformation models, designed to work simultaneously with our [Pendo source package](https://github.com/fivetran/dbt_pendo_source). A dependency on the source package is declared in this package's `packages.yml` file, so it will automatically download when you run `dbt deps`. The primary outputs of this package are described below. Intermediate models are used to create these output models.
```
[the remark this answers is not in the document]
Comment (fivetran-jamie) [reply README.md:14]: code hasn't been merged into main yet, the project will be populated once https://github.com/fivetran/dbt_pendo_source/pull/1 is merged

File: README.md [hunk 3]
```diff
@@ -27,7 +27,7 @@ This package contains transformation models, designed to work simultaneously wit
 | [pendo__guide_daily_metrics](models/pendo__guide_daily_metrics.sql)             | A daily historical timeline of the accounts and individual visitors interacting with guides via different types of actions. |
 | [pendo__feature_event](models/pendo__guide_daily_metrics.sql)             | The event stream of clicks on tagged features in Pendo. Enriched with any visitor and/or account passthrough columns, the previous feature and page that the interacted with, the application and platform the event occurred on, and information on the feature and its product area. |
```
[the remark this answers is not in the document]
Comment (fivetran-jamie) [reply README.md:28]: ah yeah should be 
```suggestion
| [pendo__feature_event](models/pendo__guide_daily_metrics.sql)             | The event stream of clicks on tagged features in Pendo. Enriched with any visitor and/or account passthrough columns, the previous feature and page that the visitor interacted with, the application and platform the event occurred on, and information on the feature and its product area. |
```

File: README.md [hunk 4]
```diff
@@ -5,8 +5,8 @@ This package models Pendo data from [Fivetran's connector](https://fivetran.com/
 
 This package enables you to better understand how users are experiencing and adopting your product. It does so by:
 - Calculating usage of features, pages, guides, and the overall product at the account and individual visitor level
-- Enhancing event stream tables with visitor and product information, along with referring pages and features in order to cultivate a customer-journey through the application
-- Creating daily activity timelines for features, pages, and guides, reflecting their adoption rates, discoverability, and efficacy at promoting more usage
+- Enhancing event stream tables with visitor and product information, referring pages, and features to cultivate a customer journey through the application
+- Creating daily activity timelines for features, pages, and guides that reflect their adoption rates, discoverability, and usage promotion efficacy
 - Directly visitors and features to determine activation rates, power-usage, and churn risk
```
[the remark this answers is not in the document]
Comment (fivetran-jamie) [reply README.md:10]: oop yeah should be something like 
```suggestion
- Directly ties visitors and features together to determine activation rates, power-usage, and churn risk
```

File: README.md [hunk 5]
```diff
@@ -27,7 +27,7 @@ This package contains transformation models, designed to work simultaneously wit
 | [pendo__guide_daily_metrics](models/pendo__guide_daily_metrics.sql)             | A daily historical timeline of the accounts and individual visitors interacting with guides via different types of actions. |
 | [pendo__feature_event](models/pendo__guide_daily_metrics.sql)             | The event stream of clicks on tagged features in Pendo. Enriched with any visitor and/or account passthrough columns, the previous feature and page that the interacted with, the application and platform the event occurred on, and information on the feature and its product area. |
 | [pendo__page_event](models/pendo__guide_daily_metrics.sql)             | The event stream of views of tagged pages in Pendo. Enriched with any visitor and/or account passthrough columns, the previous page that the interacted with, the application and platform the event occurred on, and information on the page and its product area. |
```
[the remark this answers is not in the document]
Comment (fivetran-jamie) [reply README.md:29]:
```suggestion
| [pendo__page_event](models/pendo__guide_daily_metrics.sql)             | The event stream of views of tagged pages in Pendo. Enriched with any visitor and/or account passthrough columns, the previous page that the visitor interacted with, the application and platform the event occurred on, and information on the page and its product area. |
```

File: README.md [hunk 6]
```diff
@@ -58,9 +58,9 @@ vars:
 
 ### Passthrough Columns
 
-Additionally, this package includes all source columns defined in the macros folder. We highly recommend including custom fields in this package as models now only bring in the standard fields for the `EVENT`, `FEATURE_EVENT`, `PAGE_EVENT`, `ACCOUNT_HISTORY`, and `VISITOR_HISTORY` tables.
+This package includes all of the source columns that are defined in the macros folder. We recommend including custom fields in this package because the transformation models only bring in the standard fields for the `EVENT`, `FEATURE_EVENT`, `PAGE_EVENT`, `ACCOUNT_HISTORY`, and `VISITOR_HISTORY` tables.
```
[the remark this answers is not in the document]
Comment (fivetran-jamie) [reply README.md:61]: good point, i would say "column" is the more straightforward choice of the 2

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (fivetran-chloe) [reply README.md:8]: @fivetran-jamie How about "track the customer journey"?

File: README.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (fivetran-chloe) [reply README.md:61]: Great - I'll make the change.

File: README.md [hunk 7]
```diff
@@ -58,9 +58,9 @@ vars:
...
 
-You can add more columns using our passthrough column variables. These variables allow the passthrough fields to be aliased (`alias`) and casted (`transform_sql`) if desired, although it is not required. Datatype casting is configured via a SQL snippet within the `transform_sql` key. You may add the desired SQL snippet while omitting the `as field_name` part of the casting statement - this will be dealt with by the alias attribute - and your custom passthrough fields will be casted accordingly.
+You can add more columns using our passthrough column variables. These variables allow the passthrough fields to be aliased (`alias`) and casted (`transform_sql`) if you want, although it is not required. You can configure the datatype casting using a SQL snippet within the `transform_sql` key. You may add the desired SQL snippet while omitting the `as field_name` part of the casting statement - this will be dealt with by the alias attribute - and your custom passthrough fields will be casted accordingly.
```
[the remark this answers is not in the document]
Comment (fivetran-chloe) [reply README.md]: ah, got it - I'll reword to make that clear

Comment (fivetran-chloe) [inline README.md]:
```suggestion
You can add more columns using our passthrough column variables. These variables allow the passthrough fields to be aliased (`alias`) and casted (`transform_sql`) if you want, although it is not required. You can configure the datatype casting using a SQL snippet within the `transform_sql` key. You may add the desired SQL snippet while omitting the `as field_name` part of the casting statement - we rename this column with the alias attribute - and your custom passthrough fields will be casted accordingly.
```

File: README.md [hunk 8]
```diff
@@ -58,9 +58,9 @@ vars:
...
 
-You can add more columns using our passthrough column variables. These variables allow the passthrough fields to be aliased (`alias`) and casted (`transform_sql`) if desired, although it is not required. Datatype casting is configured via a SQL snippet within the `transform_sql` key. You may add the desired SQL snippet while omitting the `as field_name` part of the casting statement - this will be dealt with by the alias attribute - and your custom passthrough fields will be casted accordingly.
+You can add more columns using our passthrough column variables. These variables allow the passthrough columns to be aliased (`alias`) and casted (`transform_sql`) if you want, although it is not required. You can configure the datatype casting using a SQL snippet within the `transform_sql` key. You may add the desired SQL snippet while omitting the `as field_name` part of the casting statement - this will be dealt with by the alias attribute - and your custom passthrough columns will be casted accordingly.
```
Comment (fivetran-chloe) [inline README.md:63]:
```suggestion
You can add more columns using our passthrough column variables. These variables allow the passthrough columns to be aliased (`alias`) and casted (`transform_sql`) if you want, although it is not required. You can configure the datatype casting using a SQL snippet within the `transform_sql` key. You may add the desired SQL snippet while omitting the `as field_name` part of the casting statement - we rename this column with the alias attribute - and your custom passthrough columns will be casted accordingly.
```

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (fivetran-jamie) [reply README.md:8]: yah that sounds good! i'll merge this PR and make the adjustment on my end

**Pull request merged**